Sources/ClientRuntime/Networking/Http/CRT/Endpoint+CRT.swift

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import enum AwsCommonRuntimeKit.EndpointProperty
+import struct Foundation.URLComponents
+import enum Smithy.ClientError
+import struct SmithyHTTPAPI.Endpoint
+import enum SmithyHTTPAPI.EndpointPropertyValue
+import struct SmithyHTTPAPI.Headers
+
+extension Endpoint {
+
+    public init(urlString: String,
+                headers: Headers = Headers(),
+                endpointProperties: [String: EndpointProperty]) throws {
```

Comment:
The initializer above is only called by `DefaultEndpointResolver` so it is moved here from `SmithyHTTPAPI`.